src/site/lab1.rst

```diff
@@ -1,8 +1,84 @@
+******
 Lab #1
+******
+
+* Feel free to use your laptop if you have it
+* I strongly encourage you to work with others in the lab
+* I will not be helping people setup IntelliJ on their laptops
+* Ensure I have recorded your completion. Failure to do so will result in a grade of 0
+* When you get stuck, do me a favour and ask those sitting around you for help
+    * I want people to get used to working together in the labs
+    * Peer teaching and peer learning is super effective
+
+Creating a Project
+==================
+
+1. Open IntelliJ
+2. Create a new project
+    * You may see a start page
+    * Or you may need to go *File -> New -> Project*
+    * Don't use the templates
+    * You should see something like this
+
+    .. image:: img/lab1_empty.png
+       :width: 500 px
+       :align: center
+
+3. Add a Java class to the project
+    * Right click *src* in the project explorer
+    * *New -> Java Class*
+    * Give your class a name, something like *Lab1*
+        * **Always start your class names with an uppercase letter**
```

Comment:
I'd probably specify, _by convention, always start..._ There's no real reason otherwise, it's only what people expect to see.